.travis.yml

```diff
@@ -20,20 +36,10 @@ install:
   - nvm version
   - node -v
   - make install_ci
+  - chmod +x ./travis/build-all-in-one-image.sh
```

Comment:
can't the exec mode be set on the file in git? it works for other scripts like license fixer